apps/dashboard/src/feature/map/hooks/useAcceptBin.ts

```
@@ -0,0 +1,23 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import api from '../../../lib/axios';
+import { Bin } from '../Bin';
+
+export const useAcceptBin = () => {
+  const queryClient = useQueryClient();
+  return useMutation<Bin, Error, number>({
+    mutationFn: async (binId: number) => {
+      try {
+        const { data } = await api.put<Bin>(`/bins/admin/${binId}/accept`, { accept: true });
+        return data;
+      } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)